occurrences/schema.py

```diff
@@ -694,7 +735,7 @@ class Input:
         group_name = graphene.String()
         extra_needs = graphene.String()
         amount_of_adult = graphene.Int()
-        study_level = StudyLevelEnum()
+        study_levels = graphene.List(StudyLevelInput)
```

Comment:
Same as [Line 519](https://github.com/City-of-Helsinki/palvelutarjotin/pull/106/files#diff-68d6b4eaea8b3fcb8aee6a0b456f07d442db6c5774a1e5ed978a684e739a2731R519)